sw/cheri/boot/boot_loader.cc

```diff
@@ -26,9 +26,27 @@ extern "C" {
 
 #define DEBUG_ELF_HEADER 0
 
+#define ARR_LEN(X) ((sizeof(X)) / (sizeof(X[0])))
+
+enum class SoftwareSlotAddress : uint32_t
+{
+	Slot1 = 0 * 10 * 1024 * 1024,
+	Slot2 = 1 * 10 * 1024 * 1024,
+	Slot3 = 2 * 10 * 1024 * 1024,
+};
+
+const SoftwareSlotAddress SoftwareSlots[] = {
+	SoftwareSlotAddress::Slot1,
+	SoftwareSlotAddress::Slot2,
+	SoftwareSlotAddress::Slot3
+};
```

Comment:
I think we don't need this additional enum class, as this causes churn later. Normal enums or just addresses directly in `SoftwareSlots` is good enough.